src/plugins/tjvendors/tjvendors/tjvendors.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  Privacy.tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Plugin\CMSPlugin;
+
+/**
+ * TJVendors Plugin.
+ *
+ * @since  2.3.4
+ */
+class PlgTjvendorsTjvendors extends CMSPlugin
+{
+	/**
+	 * Load plugin language file automatically so that it can be used inside component
+	 *
+	 * @var    boolean
+	 * @since  1.4.3
+	 */
+	protected $autoloadLanguage = true;
+
+	/**
+	 * On saving vendor data
+	 *
+	 * Method is called after vendor data is stored in the database.
+	 *
+	 * @param   array  $data  Holds the new vendor data.
+	 *
+	 * @return  void
+	 *
+	 * @since   1.4.3
+	 */
+	public function tjVendorOnAfterVendorSave($data)
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done